hardware/computemodule/cmio-display.md

```diff
@@ -0,0 +1,70 @@
+# Attaching Raspberry Pi Official 7 inch Display to the Compute Module IO Board
+
+*This document is a work in progress and is intended for advanced users.*
+
+For the display to work with the compute module, the firmware needs to be October 23rd 2015 or newer (use `vcgencmd version` to check).
+
+## Quickstart - Display Only
+
+1. Connect the display to the DISP1 port on the Compute Module IO Board through the 22W to 15W display adaptor.
+1. Connect these pins together with jumper wires:
+
+	```
+	GPIO0 - CD1_SDA
+	GPIO1 - CD1_SCL
+	```
+
+1. On the compute module, run:
+
+	```sudo wget https://goo.gl/Ah6XD5 -O /boot/dt-blob.bin```
+
+1. Make the 7 inch display as the default by adding the following line into `/boot/config.txt`:
+
+	```
+	display_default_lcd=1
+	```
+
+1. Reboot for the `dt-blob.bin` file to be read.
+
+## Quickstart - Display and Camera
+* This will enable `disp1` and `cam1` *
```

Comment:
This line looks weird in the formatted output - should the spaces between the text and the asterisks be removed?